### DISCUSSION TITLE
A proposal to add a module that expands the functionality of prefabricated chips and circuits that exclude the use of analog signals of red stone from vanilla.

### DISCUSSION THREAD

**Discussion by MasterPM724LTJT103** [MrTJP/ProjectRed#1646] [Ideas]
An analog of the comparator is needed for the ProjectRed Fabrication module, but unlike the vanilla one and from the Integration module, it must be able to freely pass through the red stone signal and interrupt the possibility of skipping it when a third-party red stone signal is applied to the side gate. I believe that if a similar analogue appears in the Fabrication and Integration modules, it will facilitate the construction of complex control and control schemes (both in manual, semi-automatic and automatic modes) and use (under the control of the user or without the participation of such at all).

**Comment by MrTJP** [comment]
Not sure what you are talking about. Analog IO are already being planned for Fabrication, and I don’t see any reason to add any more ‘analog’ gates. Feel free to give actual examples of what you are asking for, because I don’t really understand the request.
